caikit/runtime/servicers/training_management_servicer.py

```diff
@@ -131,7 +131,7 @@ def _get_model_future(training_id: str, operation: str):
         Wrapped here so that we only catch errors directly in the `trainer.get_model_future` call
         """
         try:
-            return MODEL_MANAGER.get_model_future(training_id=training_id)
+            return MODEL_MANAGER.get_model_future(training_id)
```

Comment:
Why remove the keyword arg here? I think this may have been residual from them being merged into `get_model_future`, but I'd rather keep the explicit kwarg here